content/docs/guides/blackbox-exporter.md

```diff
@@ -0,0 +1,151 @@
+
+```yaml
+scrape_configs:
+- job_name: blackbox
+  metrics_path: /probe
+  scrape_interval: 5s
+  params:
+    module: [http_2xx]
+  static_configs:
+  - targets:
+    - localhost:2112/health
+  relabel_configs:
+  - source_labels: [__address__]
+    target_label: __param_target
+  - source_labels: [__param_target]
+    target_label: instance
+  - target_label: __address__
+    replacement: localhost:9115
+```
+
+This instructs Prometheus to scrape the Blackbox Exporter's `/probe` endpoint and passes the `targets` list, which includes the Python web server running on port 2112, to the Blackbox Exporter. With this configuration in place, start Prometheus:
+
+```shell
+./prometheus
+```
+
+## Installing, configuring, and running the Blackbox Exporter
+
+Like Prometheus, the Blackbox Exporter is a single static binary that you can install via tarball. [Download the latest release](/download#blackbox_exporter) for your platform and untar it:
+
+```bash
+wget https://github.com/prometheus/blackbox_exporter/releases/download/v*/blackbox_exporter-*.*-amd64.tar.gz
+tar xvfz blackbox_exporter-*.*-amd64.tar.gz
+cd blackbox_exporter-*.*-amd64
+```
+
+The untarred folder contains a `blackbox.yml` file for configuring the exporter. Delete the current contents and replace it with this:
+
+```yaml
+modules:
+  http_2xx:
+    prober: http
+    timeout: 5s
+    http:
+      method: GET
+```
+
+With this configuration, the Blackbox Exporter will use the built-in `http_2xx` module, which periodically probes HTTP endpoints (the interval is set in your Prometheus configuration).
+
+NOTE: In the Blackbox Exporter configuration, you don't need to specify host or port information for the specific endpoints that you want to probe, only which modules you'd like to use. Host and port information is specifed in the [Prometheus configuration](#prometheus-configuration).
+
+To start up the Blackbox Exporter on port 9115 using the `blackbox.yml` configuration:
+
+```shell
+./blackbox_exporter
+```
+
+## Exploring Blackbox Exporter probes
+
+Navigate to http://localhost:9115 in your browser to see the Blackbox Exporter's web interface. Under the **Recent Probes** section you'll see results from the exporter's most recent probes as a table. You should see entries like this:
+
+Module | Target | Result | Debug
+:------|:-------|:-------|:-----
+`http_2xx` | `localhost:2112/health` | Success | Logs
+`http_2xx` | `localhost:2112/health` | Success | Logs
+`http_2xx` | `localhost:2112/health` | Success | Logs
+
+In the Prometheus [expression browser](/docs/visualization/browser), accessible at http://localhost:9090/graph, you can a variety of Blackbox-Exporter-specific metrics, which have the `probe_` prefix, for example, the [`probe_success{instance="localhost:2112/health",job="blackbox"}`](http://localhost:9090/graph?g0.range_input=1h&g0.expr=probe_success{instance%3D%localhost%3A2112%2Fhealth%22}), which indicates whether the most recent probe was successful.
```

Comment:
You need to talk more about probe_success, many users miss the importance of this

Maybe setup an alert?


I'd also mention the debug URL parameter.